README.md

```diff
@@ -16,12 +16,18 @@
 
 Check the new [Guide for Nuxt Modules](https://v3.nuxtjs.org/docs/advanced/modules/) before starting with module-builder.
 
+### Requirements
+
+For a user to use a module generated from module-builder, it's recommended they have:
+- Node.js >= 14.x. _Latest Node LTS preferred_
```

Comment:
We would ensure this by adding `engines` field to `package.json` of starter template and check it directly to warn module author wdyt? Since this guide is for module authors but Node.js requirement also applies for users of module